source/common/upstream/load_balancer_impl.h

```diff
@@ -757,10 +757,10 @@ class SubsetSelectorImpl : public SubsetSelector {
 
 private:
   const std::set<std::string> selector_keys_;
+  const std::set<std::string> fallback_keys_subset_;
```

Comment:
Why was this field moved to the end?

Reply:
I wasn't moved down, it was moved up above an enum

Reply:
Ah... misread the change. That makes sense.
Can you please add a relevant comment (before the enum) similar to other PRs?

Reply:
Done